src/game/default/g_types.h

```diff
@@ -817,6 +825,7 @@ typedef struct {
 
 	const g_item_t *item; // for bonus items
 
+	_Bool held_grenade;	// is this entity a nade that is being held currently?
```

Comment:
There's probably a way to condense all of this state a bit, but if you can confirm that these fields are all in use right now, I'm happy to merge as-is.